libraries/resource_providers/api1800/c7000/hypervisor_cluster_profile_provider.rb

```diff
@@ -0,0 +1,32 @@
+# (c) Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+module OneviewCookbook
+  module API1800
+    module C7000
+      # Hypervisor Cluster Profile API1800 provider
+      class HypervisorClusterProfileProvider < API1600::C7000::HypervisorClusterProfileProvider
+        def delete
```

Comment:
no need to add this method here.. Simply inheriting from API1600 will get that method also. Remove it here.

Reply:
Done